demos/tests/args.py

```diff
@@ -73,6 +73,27 @@ def resolve(self, context):
         return str(seq_dir / name_format)
 
 
+class DataOrigFileNamesArg:
```

Comment:
Why do you need this as a separate class? Just merge it into `DataDirectoryOrigFileNamesArg`.